### PR TITLE
Fix circular imports

### PR DESCRIPTION
**Pull request by mauritsvanrees**
Hopefully fixes issue #95.

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
This brings landscape to 99 percent: https://landscape.io/github/zestsoftware/zest.releaser/86

@reinout Can you have a quick look? After that: shall we do a release?

File: README.rst [hunk 1]
```diff
@@ -38,11 +38,15 @@ First the three most important links:
 - The code is at `github.com/zestsoftware/zest.releaser
   <https://github.com/zestsoftware/zest.releaser>`_.
 
-And... we're automatically being tested by Travis:
+And... we're automatically being tested by Travis and Landscape:
```
Comment (reinout) [inline README.rst]: Yeah! Good one to add.

**Comment by reinout** [comment]
99%! Nice.

Yes, feel free to release. Give me a ping when finished :-)

**Comment by mauritsvanrees** [comment]
Ping. :-)

```
INFO: This package is registered on PyPI.
Register and upload to pypi (Y/n)? 
INFO: Running: twine upload dist/* -r pypi
Uploading distributions to https://pypi.python.org/pypi
Uploading zest.releaser-4.0-py2-none-any.whl
Uploading zest.releaser-4.0.tar.gz
```

**Comment by mauritsvanrees** [comment]
Also, I have removed all branches except master. All the others were merged, had no changes, or were otherwise not relevant anymore.

**Pull request merged**